content/coronavirus_landing_page.yml

```
@@ -3,16 +3,18 @@ content:
   meta_description: "Find information on coronavirus, including guidance, support, announcements and statistics."
   page_header: "Coronavirus (COVID&#8209;19)"
   header_section:
-    title: "Coronavirus restrictions remain in place across the country, including for people who have been vaccinated. In England:"
+    changes_2021_07_19: true
+    title: "Coronavirus remains a serious health risk. It’s important to stay cautious and help protect yourself and others. In England:"
     intro: |
-      - You can meet indoors in a group of up to 6 people or a group of any size from 2 households 
-      - You can meet outside in a group of up to 30 people
-      - Work from home if you can and travel safely
-      - If you have symptoms get a test and stay at&nbsp;home
+      - <a href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/">Get vaccinated if you are 18 or over</a>
+      - Meet outdoors where possible and let fresh air into indoor spaces
+      - Wear a face covering in crowded areas, such as on public transport
+      - <a href="https://www.nhs.uk/conditions/coronavirus-covid-19/testing/regular-rapid-coronavirus-tests-if-you-do-not-have-symptoms/">Take regular lateral flow tests</a>, even if you do not have symptoms
+      - You must self-isolate and <a href="https://www.gov.uk/get-coronavirus-test">take a PCR test</a> if you have symptoms
```

```suggestion
      - [Get vaccinated if you are 18 or over](https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/)
      - Meet outdoors where possible and let fresh air into indoor spaces
      - Wear a face covering in crowded areas, such as on public transport
      - [Take regular lateral flow tests](https://www.nhs.uk/conditions/coronavirus-covid-19/testing/regular-rapid-coronavirus-tests-if-you-do-not-have-symptoms/), even if you do not have symptoms
      - You must self-isolate and [take a PCR test](/get-coronavirus-test) if you have symptoms
```

We can set these as markdown links

---

Ah of course. I'm happy for someone else to make these changes if necessary.